tinydrive/tinydrive-api/standalone.md

```diff
@@ -55,6 +78,34 @@ The modification date for the file in ISO 8601 format for example `2019-02-24T15
 
 {% include live-demo.html id="drive-standalone-pick" type="tinydrive" %}
 
+## `tinydrive.controls.pick`
+
+The `tinydrive.controls.pick` method allows users to pick files from {{site.cloudfilemanager}} and get the metadata of those files returned in a callback. For a complete list of available settings to pass into this method, see: [Picker settings](#pickersettings). A [Control Object](#controlobject) is returned for controlling the {{site.cloudfilemanager}} lifecycle, which is not handled by {{site.cloudfilemanager}} when using the `controls` APIs.
+
+### File picker result format
+
+The `tinydrive.controls.pick` method will return the selected value to the provided callback as an array of files with the following properties.
+
+#### name
+
+The name of the selected file. For example: `my.jpg`.
+
+#### size
+
+The size in bytes of the selected file.
+
+#### url
+
+The URL for the selected file. The URL will be in the following format: `https://drive.tiny.cloud/1/<your api key>/<file uuid>`
+
+#### mdate
+
+The modification date for the file in ISO 8601 format. For example: `2021-02-24T15:00:00Z`
+
+### Interactive example: Using `tinydrive.controls.pick`
+
+{% include live-demo.html id="drive-standalone-pick-controls" type="tinydrive" %}
+
 ## `tinydrive.browse`
 
 The `tinydrive.browse` method enables you to browse your files stored in {{site.cloudfilemanager}} but without the possibility to pick them to be inserted. This might be useful if you want to use {{site.cloudfilemanager}} as a generic file manager. It returns a promise but the promise will only resolve when the {{site.cloudfilemanager}} dialog is closed by using the close button. For a complete list of available settings to pass in to this method check the [Picker settings](#pickersettings) section in this page.
```

Comment:
Should this have also been updated to say `allows users`? It was done for `pick` but not here.